src/Installer.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App;
+
+use Composer\Script\Event;
+
+class Installer
+{
+    public static function postInstall(Event $event): void
```

Comment:
```suggestion
    public static function postInstall(Event $event = null): void
```
Because we don't use this event